lib/authc/OAuthBasicExchangeAuthenticator.js

```diff
@@ -47,6 +47,7 @@ OAuthBasicExchangeAuthenticator.prototype.authenticate = function authenticate(c
 
         result.forApiKey = apiKey;
         result.application = self.application;
+        result.expiration = self.ttl;
```

Comment:
As pointed out in your other comment, I also see how this should be renamed to `ttl` instead.